### DISCUSSION TITLE
Implementing internationalization (i18n) in Next.js for a project

### DISCUSSION THREAD

**Discussion by Sathyaraj-dev** [vercel/next.js#67561] [Help]
### Summary

Hey everyone,

I'm looking for recommendations on implementing internationalization (i18n) in Next.js for a project that will support both English and Arabic. Could you please suggest a good npm package for this purpose?"

Would you like me to suggest some npm packages for internationalization in Next.js that support multiple languages?

Thanks

**Comment by BorhanSaflo** [comment]
[next-int](https://next-intl-docs.vercel.app/docs/next-13) is an excellent package for internationalization in Next.js. 

The [internationalization](https://nextjs.org/docs/app/building-your-application/routing/internationalization) section of the Next.js documentation is also useful.

**Reply by Sathyaraj-dev** [reply]
Hi BorhanSaflo,

Thanks for the reply. I tried this [internationalization](https://nextjs.org/docs/app/building-your-application/routing/internationalization) but, still something is missing from this. So I will try the [next-int.](https://next-intl-docs.vercel.app/docs/next-13).
